packages/core/src/api/app/AppBuilder.tsx

```diff
@@ -86,6 +87,10 @@ export default class AppBuilder {
       }
     }
 
+    routes.push(
+      <Route key={'login'} path={'/login'} component={LoginPage} exact />,
```

Comment:
The intention is to not have any hardcoded routes here at all. I think this would do better as a manual `<Route>` element in the app, as in `packages/app`. I'm fine with merging this as is though since we don't have a clear pattern for it yet.